compliance_suite/job_runner.py

```diff
@@ -126,6 +126,24 @@ def tag_matcher(
                 return True
         return False
 
+    def version_matcher(
+            self,
+            versions: List[str]
+    ) -> bool:
+        """ Matches the user provided spec version with the YAML test versions.
+         Skips the test if versions are not matched.
+
+        Args:
+            versions: The versions defined for a YAML test
+
+        Returns:
+            True, if the versions match. Otherwise, false.
+        """
+
+        if self.version in versions:
+            return True
+        return False
```

Comment:
Note that you could use Python's faux ternary operator here:

```python
True if self.version in versions else False
```

But that's just FYI, no need to change ;-)